lib/services/android/gradle-build-service.ts

```diff
@@ -0,0 +1,30 @@
+import { attachAwaitDetach } from "../../common/helpers";
+import * as constants from "../../constants";
+import { EventEmitter } from "events";
+
+export class GradleBuildService extends EventEmitter implements IGradleBuildService {
+	constructor(
+		private $childProcess: IChildProcess,
+		private $gradleBuildArgsService: IGradleBuildArgsService,
+		private $gradleCommandService: IGradleCommandService,
+	) { super(); }
+
+	public async buildProject(projectRoot: string, buildConfig: IAndroidBuildConfig): Promise<void> {
+		const buildTaskArgs = this.$gradleBuildArgsService.getBuildTaskArgs(buildConfig);
+		const spawnOptions = { emitOptions: { eventName: constants.BUILD_OUTPUT_EVENT_NAME }, throwError: true };
+		const gradleCommandOptions = { cwd: projectRoot, message: "Gradle build...", stdio: buildConfig.buildOutputStdio, spawnOptions };
```

Comment:
Previously there was a default value for the stdio - inherit. Does it still default to that down the execution line?

Reply:
We still have a default value - https://github.com/NativeScript/nativescript-cli/pull/4537/files#diff-235c99870cec07a0b3afcf34af51be12R10

Reply:
Saw it in executeCommand :)